Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators_test.py

```diff
@@ -244,3 +244,17 @@ def test_create_values_out_dict_4(self):
                 iocs_txt_json = json.load(iocs_txt_f)
                 for line in text_out.split('\n'):
                     assert line in iocs_txt_json
+
+    @pytest.mark.validate_basic_authentication
+    def test_validate_basic_authentication(self):
+        """Test Authentication"""
+        from ExportIndicators import validate_basic_authentication
+        username, password = 'user', 'pwd'
+        with open('ExportIndicators_test/TestHelperFunctions/authentication_test_data.json', 'r') as f:
```

Comment:
I am not sure why you are doing this. Having the test data in a file is not necessary and actually just makes it harder to understand what is being tested. You can just pass the header that you want to test in the code.